docusaurus.config.js

```diff
@@ -44,12 +44,12 @@ module.exports = {
     ],
   ],
   plugins: [
-    [require.resolve('@cmfcmf/docusaurus-search-local'), {
-        indexDocs: true,
-        docsRouteBasePath: '/',
-        indexDocSidebarParentCategories: 3,
-        indexPages: false,
-        language: "en"
-    }]
+    //[require.resolve('@cmfcmf/docusaurus-search-local'), {
```

Comment:
please add here a description of why it is commented out

Reply:
Fixed in commit: 6190797accdad166b10e8d510999e070fd34edae